schemas/example_evaluation_results.yml

```diff
@@ -0,0 +1,34 @@
+name: Open Source Security Baseline
```

Comment:
This example file was validated by running: `cue vet schemas/layer-4.cue schemas/example_evaluation_results.yml -d '#FrameworkEvaluation'`